Mechty-learn-backend/Mechty-learn-backend/Program.cs

```diff
@@ -2,6 +2,7 @@
 using Mechty_learn_backend.Data;
 using Mechty_learn_backend.Models;
 using Mechty_learn_backend.Repositories;
+using Mechty_learn_backend.Repositories.EducationRepositories;
```

Comment:
Just a stylistic suggestion: "MechtyLearn" would be a better namespace name. Consider refactoring it in the future. I think you need to "backend" tag at the end. It's obvious after a certain point what code runs on the backend and which isn't.